src/_P022_PCA9685.ino

```diff
@@ -294,19 +317,19 @@ uint8_t Plugin_022_readRegister(int i2cAddress, int regAddress) {
 //********************************************************************************
 // PCA9685 write
 //********************************************************************************
-void Plugin_022_Off(int port, int pin)
+void Plugin_022_Off(int address, int pin)
 {
-  Plugin_022_Write(port, pin, 0);
+  Plugin_022_Write(address, pin, 0);
 }
 
-void Plugin_022_On(int port, int pin)
+void Plugin_022_On(int address, int pin)
 {
-  Plugin_022_Write(port, pin, PCA9685_MAX_PWM);
+  Plugin_022_Write(address, pin, PCA9685_MAX_PWM);
 }
 
-void Plugin_022_Write(int port, int Par1, int Par2)
+void Plugin_022_Write(int address, int Par1, int Par2)
```

Comment:
Could you please change these 2 I2C communication functions to use the new functions in https://github.com/letscontrolit/ESPEasy/blob/mega/src/I2C.ino  ?
This makes it easier later to globally change I2C related code.

Reply:
I have tried it, but i have found some issues. I don't have time to investigate.

Reply:
OK, maybe we could discuss later on these issues.
I would like to hear about them.